cmd/pfcpctl/main.go

```diff
@@ -0,0 +1,155 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright 2022 Open Networking Foundation
+
+package main
+
+import (
+	"context"
+	"os"
+
+	pb "github.com/omec-project/pfcpsim/api"
+	"github.com/pborman/getopt/v2"
+	log "github.com/sirupsen/logrus"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials/insecure"
+)
+
+const (
+	defaultAddress           = ":54321"
+	defaultRemotePeerAddress = "127.0.0.1"
+	defaultUEAddressPool     = "17.0.0.0/24"
+
+	// values for UP4 environment
+	//defaultN3Address     = "140.0.0.1"
+	//defaultNodeBAddress  = "140.0.100.1"
+
+	// Values for mock-up4 environment
+	defaultNodeBAddress = "198.18.0.10"
```

Comment:
IMO we shouldn't assume any default IP addresses/values.

Reply:
let's make it mandatory part of `Configure`

Reply:
Done.